package.json

```diff
@@ -21,6 +21,7 @@
     "cartocolor": "^4.0.0",
     "earcut": "^2.1.2",
     "jsep": "^0.3.2",
+    "lodash": "^4.17.5",
```

Comment:
any chance of not using loadash? IMO we should create an `utils` folder

Reply:
I think we can use it right now in order to save time, but for the alpha or beta release we could improve the bundle size with this proposal if required.

Reply:
I have my doubts too, but I wanted some of their utils in the past and now I think it would probably save us some time. I would try to use native functions as much as possible, but I think it's fine for now.